cylc/flow/cfgspec/globalcfg.py

```diff
@@ -434,6 +434,8 @@
                 accepts up to 236 characters.
             ''')
             Conf('owner', VDR.V_STRING)
+        with Conf('localhost', meta=Platform):
+            Conf('remote hosts', VDR.V_STRING_LIST, ['localhost'])
```

Comment:
Note to other reviewers: This is configuration inheritance which allows us to define defaults for specific cases (e.g. the `localhost` platforms) in a way that avoids duplication and gets auto-documented nicely.

See also https://github.com/cylc/cylc-flow/pull/3630.